### ISSUE TITLE
Helper function to get grid and display options

### ISSUE THREAD

**Issue by JohnOmernik** [quantopian/qgrid#121]
Having a simple function to return the options that are currently set with qgrid would be handy. (Perhaps this would be a good place to document the options too, and then have it return a df that is displayed in qgrid! :)

**Comment by TimShawver** [comment]
Good idea, thanks

**Comment by TimShawver** [comment]
In the new qgrid 1.0 beta, the options for the currently displayed `QgridWidget` are available via attributes on the `QgridWidget` object that have the same name as the kwargs you passed in to create the `QgridWidget`.  See the updated API docs here for those attributes: http://qgrid.readthedocs.io/en/latest/#qgrid.QgridWidget.grid_options

To retrieve the df that is displayed in qgrid (i.e. one which will reflect sorting/filtering, as well as edits) then you'll want to use the `get_changed_df` method on the `QgridWidget` instance.

Here's the documentation for the `get_changed_df` method: http://qgrid.readthedocs.io/en/latest/#qgrid.QgridWidget.get_changed_df

Basically you'll create a QgridWidget instance (either directly or by calling show_grid), and then call `get_changed_df` on that instance.  Something like this:
```
# step 1 - first create the grid
qgrid_widget = qgrid.show_grid(df)

# step 2 - make your filtering/sorting changes and cell edits

# step 3 - in a new cell, get your changed DataFrame back
qgrid_widget.get_changed_df()
```

I *think* this resolves your issue but please let me know if I misunderstood your suggestion @JohnOmernik

**Comment by stef30360** [comment]
#### Hi, 
#### Actually I work with Qgrid and I have two questions:

1. I don't understand, why can't I use grid_options ?
##### without grid_options
<img width="1104" alt="capture d ecran 2018-01-01 a 13 48 35" src="https://user-images.githubusercontent.com/11021144/34467744-812f97ee-eefa-11e7-898b-f15fab35cb1a.png">

##### with grid_options, the dataframe isn't display ?
<img width="1105" alt="capture d ecran 2018-01-01 a 13 43 08" src="https://user-images.githubusercontent.com/11021144/34467730-1307a432-eefa-11e7-9847-31018243ce0e.png">

2. How can I display all 24 rows of my  dataframe ?
thks
SR

**Comment by TimShawver** [comment]
@stef30360 it looks like I forgot to update part of the API docs to add some new default options for grid_options in version 1.0.  Sorry about the confusion.  Here's the new default dictionary for grid_options (I haven't updated the API docs yet but I'll do that soon):
```
{
            'fullWidthRows': True,
            'syncColumnCellResize': True,
            'forceFitColumns': True,
            'defaultColumnWidth': 150,
            'rowHeight': 28,
            'enableColumnReorder': False,
            'enableTextSelectionOnCells': True,
            'editable': True,
            'autoEdit': False,
            'explicitInitialization': True,
            'maxVisibleRows': 15,
            'minVisibleRows': 8
        }
```

The last two options `maxVisibleRows` and `minVisibleRows` are options that I added specifically for qgrid.  In other words you won't see them in the SlickGrid documentation.

In your case it sounds like you should set `maxVisibleRows` to `28`.  To do that without having to pass the entire dictionary of default options you can use `show_grid` to generate the widget as follows:
```
qgrid_widget = qgrid.show_grid(resultat, grid_options={'maxVisibleRows': 28}, show_toolbar=True)
qgrid_widget
```

**Comment by stef30360** [comment]
thks  u ! It's work. Do you have already use   ipywidget slider in interaction with Qgrid ?
regards 
SR

**Comment by TimShawver** [comment]
Yea I just made one, see my comment on your other issue: https://github.com/quantopian/qgrid/issues/156#issuecomment-354684651

**Comment by Meterology** [comment]
how to get_changed_data() import in csv file

**Comment by Meterology** [comment]
https://github.com/Meterology/Qgrid/blob/master/cgmlilab_iiserb_qgrid.pdf
